po/extra/fr.po

```diff
@@ -0,0 +1,72 @@
+# French translations for extra package.
+# Copyright (C) 2019 THE extra'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the extra package.
+# NathanBnm, 2019.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-06-22 16:07+0200\n"
+"PO-Revision-Date: 2019-06-22 16:01+0200\n"
+"Last-Translator: NathaBnm\n"
+"Language-Team: none\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: data/com.github.ryonakano.spreadsheet.desktop.in:3
+#: data/com.github.ryonakano.spreadsheet.appdata.xml.in:7
+msgid "Spreadsheet"
+msgstr "Spreadsheet"
+
+#: data/com.github.ryonakano.spreadsheet.desktop.in:4
+msgid "Spreadsheet App"
+msgstr "Application Spreadsheet"
+
+#: data/com.github.ryonakano.spreadsheet.desktop.in:5
+#: data/com.github.ryonakano.spreadsheet.appdata.xml.in:8
+msgid "Create simple and beautiful spreadsheets"
+msgstr "Créez des feuilles de calcul simples et belles"
+
+#: data/com.github.ryonakano.spreadsheet.desktop.in:8
+msgid "com.github.ryonakano.spreadsheet"
+msgstr "com.github.ryonakano.spreadsheet"
+
+#: data/com.github.ryonakano.spreadsheet.desktop.in:11
+msgid "Office;Book;"
+msgstr "Bureautique;Livre;"
```

Comment:
I don't know if categories like that should be translated?

Reply:
I think yes as they are translated in the AppCenter. @ryonakano any idea?

Reply:
I think so too. As far as I glanced at some apps of elementary, @NathanBnm seems to translate this kind of strings, so I think we can translate this string too and keep consistency in UX with elementary apps.

Reply:
And this is not necessary and in my opinion, but  in elementary OS some languages like Traditional Chinese or Japanese (yes, one of the Japanese translators is me :smile:) translates this kinds of string like this (sorry for Japanese example):

```
Office;Book;オフィス;ブック;
```

I put original English strings first, and after that put translated Japanese string. I think this translation way allows users to search the app not only in translated string, but also in original English string.

Reply:
@ryonakano in the French version of the AppCenter there is an "Office" category which is translated "Bureautique" so I think it should work. However the "Book" category doesn't exist in AppCenter.

Reply:
![Screenshot from 2019-06-25 07-15-05](https://user-images.githubusercontent.com/26003928/60055610-1f807680-9719-11e9-8914-95ddef510158.png)

@NathanBnm Do you mean these category in the Home page of AppCenter? Actually, these categories and this string (`Office;Book;`) is not relevant.

How AppCenter categorize apps to these categories is checking `Categories:` line in .desktop files. For example, this app has the following categories:

https://github.com/ryonakano/Spreadsheet/blob/76473f6e8006fa355aa5c5d648784b9e3e55918f/data/com.github.ryonakano.spreadsheet.desktop.in#L5

In this case, this app should be categorized to "Office", because if `Categories:` line of an app has "Office", "Presentation", "Publishing", "Spreadsheet", or "WordProcessor", AppCenter will categorize to "Office":

https://github.com/elementary/appcenter/blob/83bca8521a7b4518b39b009c0d4e0f1b49db0662/src/Widgets/CategoryFlowBox.vala#L108-L113

On the other hand, this string (`Office;Book;`) is at `Keywords:` line in .desktop file:

https://github.com/ryonakano/Spreadsheet/blob/76473f6e8006fa355aa5c5d648784b9e3e55918f/data/com.github.ryonakano.spreadsheet.desktop.in#L10

This string is used to search the app, so any string is OK. And the reason why I choose "Book" is that we often call an Excel document as a "book".